torrt/utils.py

```diff
@@ -82,6 +83,7 @@ def request(
         :param allow_redirects:
         :param cookies:
         :param headers: Additional headers
+        :param proxies: Proxies to specific request
```

Comment:
I wonder why do you use `to` preposition here and in similar cases above. Shouldn't It be `for a` instead?